src/main/java/ru/sbtqa/tag/allurehelper/Type.java

```diff
@@ -1,25 +1,37 @@
 package ru.sbtqa.tag.allurehelper;
 
+/**
+ * Enum for a attachment to addAttachment method in ParamsHelper class.
+ *
+ * @see ru.sbtqa.tag.allurehelper.ParamsHelper *
+ */
+
 public enum Type {
 
-    TEXT("text/plain"),
-    XML("application/xml"),
-    HTML("text/html"),
-    PNG("image/png"),
-    JPEG("image/jpeg"),
-    CSV("text/csv"),
-    VIDEO("video/webm"),
-    URI("text/uri-list"),
-    TARGZIP("application/x-gtar");
+    TEXT("text/plain", ".txt"),
+    XML("application/xml", ".xml"),
+    HTML("text/html", ".html"),
+    PNG("image/png", ".png"),
+    JPEG("image/jpeg", ".jpg"),
+    CSV("text/csv", ".csv"),
+    VIDEO("video/webm", ".webm"),
+    URI("text/uri-list", ".uri"),
+    TARGZIP("application/x-gtar", ".gtar");
 
     private final String type;
+    private final String extension;
 
-    private Type(final String type) {
+    Type(final String type, final String extension) {
         this.type = type;
+        this.extension = extension;
     }
 
     @Override
     public String toString() {
         return type;
```

Comment:
maybe now we also want to have an extension returning in toString method and individual getter for type